app/assets/src/components/views/TaxonTreeVis.jsx

```diff
@@ -352,6 +352,7 @@ class TaxonTreeVis extends React.Component {
           ref={container => {
             this.treeContainer = container;
           }}
+          style={{ overflowX: "scroll" }}
```

Comment:
I added this inline because there was no associated CSS file that I could find, and it seems likely this is will be a single instance

Reply:
Search for `taxon_tree_vis.scss`. It is old code, so still loaded in `loader.scss`.

Reply:
thanks!